### DISCUSSION TITLE
Can I update package @sveltejs/kit?

### DISCUSSION THREAD

**Discussion by lightr4in** [skeletonlabs/skeleton#1811] [Q&A] [answered]
Hello,

I've just created a new skeleton project and noticed the sveltekit version used which is `1.5`. The most recent `@sveltejs/kit` package version is [1.22.4](https://github.com/sveltejs/kit/releases/tag/%40sveltejs%2Fkit%401.22.4).

Is it generally safe to update this package after creating a bare skeleton project?

Thank you.

**Comment by endigo9740** [comment, marked as answer]
@lightr4in of course. We pin the version to ensure an emerging version doesn't break compatibility, but you are welcome to update. If you run into any issue just revert and let us know. Thanks!
